scalikejdbc-core/src/main/scala/scalikejdbc/SQLInterpolationString.scala

```diff
@@ -47,13 +47,15 @@ class SQLInterpolationString(private val s: StringContext) extends AnyVal {
           .map {
             case SQLSyntax(s, _)                           => s
             case SQLSyntaxParameterBinder(SQLSyntax(s, _)) => s
+            case TypedParameterBinder(_, dbType, _)        => "? :: " + dbType
```

Comment:
This can result in an SQL injection vulnerability in some use cases. We are unable to have this change.

Reply:
I changed the type of dbType to SQLSyntax